### ISSUE TITLE
[Feature] small data overfitting check

### ISSUE THREAD

**Issue by BigGold0202** [open-mmlab/mmdetection3d#2643]
### What is the feature?

I'm new to 3D object detection, and I'm training a customized model using nuscenes dataset. But the loss refuses to decrease after serveral epochs. I want to check whether my model is correct by overfitting the model to a small batch of data. However, it seems that I can achieve this only by filtering the files of nuscenes dataset and creating a new dataset. Is it possible to achieve this by modifying the dataset configs?

**Comment by Xiangxu-0103** [comment]
You can set `indices=xxx` in `train_dataloader.dataset` to use the first `xxx` samples of the dataset.

**Comment by BigGold0202** [comment]
> You can set `indices=xxx` in `train_dataloader.dataset` to use the first `xxx` samples of the dataset.

Thank you so much! I trid to change the dataloader sampler to pytorch SubsetRandomSampler which doesn't work out well. I solved this by building a new version of Nuscenes with only one sample in it, but still has some bugs. This modification in the config really helps!
